package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
ok - thanks for clarification. I feel like we get this wrong a lot in other libraries and just wanted to clarify the options again here...

This is a good summary of when to use each:
https://yarnpkg.com/lang/en/docs/dependency-types/

And agree w/ you now that patternfly and react should be peerDep (as they are installed also by the consumer). 

And to your point about making react-bootstrap a hard dependency, I agree (as it would be odd to always ask the consumer to install it manually himself). I guess I was looking at this from the other side of cases we may not necessarily be consuming react-bootstrap in some apps vs others (thus the reason to mark it optional and reduce the build size). My builds in OCP run `npm install --production` to prune non dev dependencies, but would include this package (even if not using it directly in my components yet). But say in the future we were to pull in something like a d3 / react library, like [this](https://vx-demo.now.sh/docs). Would we prefer it optional?

FWIW, I'm OK w/ leaving it in. Build size of runtime deps is ultimately not a huge deal (just increased image build a bit). Webpack2 / tree-shaking takes care of runtime optimizations on the client...